src/tokens/borders/_borders.scss

```diff
@@ -1,6 +1,8 @@
 @forward '../colors/colors';
 
 :root {
+    --selector-border-radius: 2px;
+    --selector-border-radius_big: 4px;
```

Comment:
We can go further and always manage all border-radius with variables. I propose:
```
--radius: 4px;
--radius_selector: 2px;
--radius_rounded: 2rem;
```

This naming is not perfect because it mixes the style `rounded` and the usage `selector`, but I hope it will be easier to understand when to use each variables.